### PR TITLE
Routes

### PR DESCRIPTION
**Pull request by Leandro-Barretoo**
In this PR I completed the rest of the website.
This includes:
- Home page;
- Calculator page;
- Quote page;

I used React Router to link all the pages together forming a SPA.
Thank you for the review.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @Leandro-Barretoo 👋,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        <Link to="/math__magicians" className="Visited">Math Magicians</Link>
+      </span>
+      <ul className="link-container">
+        <li className="links">
+          <Link to="/math__magicians" className="Visited">Home</Link>
+        </li>
+        <li className="links">
+          <Link to="/math__magicians/calculator" className="Visited">Calculator</Link>
+        </li>
+        <li className="links">
```
Comment (mikenath223) [inline src/components/Navigation.js]: - [x] Kindly move the inline styles to the stylesheet. It keeps the project neat and professional.

Comment (Leandro-Barretoo) [reply src/components/Navigation.js]: Hello, I have put the styles this way cause in the requirements its says to use React basic styling.
So I wanted to include every way you can add style. That's why I have stylesheets, inline and style variables.
But the fix is done, I hope it is good now.
Thank you for the review.

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**